core/iwasm/aot/arch/aot_reloc_xtensa.c

```diff
@@ -263,10 +263,17 @@ apply_relocation(AOTModule *module, uint8 *target_section_addr,
             imm16 = (int16)(relative_offset >> 2);
 
             /* write back the imm16 to the l32r instruction */
+#if __GNUC__ >= 9
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Waddress-of-packed-member"
```

Comment:
@casaroli Could you add comments about the code? It is not easy to understand what the flag means here.

Reply:
added on fa2363ee05e3ae65a039f5bba2e12b599c84b2f3